includes/kaltura_derivatives.inc

```diff
@@ -56,7 +96,14 @@ function islandora_video_streaming_kaltura_upload($object, $force = false) {
         $failureMessage['messages'][0]['message_sub']['@filename'] = $temp_name;
         return $failureMessage;
       }
-      return islandora_video_add_datastream($object, 'KALTURA_STREAM', $temp_name);
+      
+      $arr = islandora_video_add_datastream($object, 'KALTURA_STREAM', $temp_name);
```

Comment:
This is meaningless variable naming.  Perhaps `$result` or `$resultMessage` or similar?